cli/main.go

```diff
@@ -43,11 +43,10 @@ func buildVersionString() string {
 			fmt.Fprintf(versionsWriter, "\t%s\n", image)
 		}
 	}
-	fmt.Fprint(versionsWriter, "\n")
-	fmt.Fprintf(versionsWriter, "reference values for %s platform:\n", platforms.AKSCloudHypervisorSNP.String())
-	fmt.Fprintf(versionsWriter, "\truntime handler:\tcontrast-cc-%s\n", manifest.TrustedMeasurement[:32])
-	fmt.Fprintf(versionsWriter, "\tlaunch digest:\t%s\n", manifest.TrustedMeasurement)
-	fmt.Fprintf(versionsWriter, "\tgenpolicy version:\t%s\n", constants.GenpolicyVersion)
+	if refValues, err := json.MarshalIndent(manifest.EmbeddedReferenceValues(), "\t", "  "); err == nil {
```

Comment:
```suggestion
	if refValues, err := json.MarshalIndent(manifest.EmbeddedReferenceValues(), "\t\t\t\t", "  "); err == nil {
```
... to align nicely: https://go.dev/play/p/6GY8oxwZTsd.

Reply:
Huh?

I tested it so that it aligns:
```
[14:32] moritzs:☸ msanft-contrast in contrast @ msanft/tdx-trusted-measurement-embedding [$!] took 1m15s 
❯ nix run .#contrast.cli -- -v
warning: Git tree '/home/moritzs/dev/edgeless/contrast' is dirty
contrast version 0.9.0-pre

container image versions:
embedded reference values:    {
                                "AKS": {
                                  "SNP": {
                                    "MinimumTCB": {
                                      "BootloaderVersion": 3,
                                      "TEEVersion": 0,
                                      "SNPVersion": 8,
                                      "MicrocodeVersion": 115
                                    }
                                  },
                                  "TrustedMeasurement": "e56b58bdc033c24c28bac08e6afe094e248824cfaaf885aad7c2c832a5c3672645bc83b0e1bfec309b00fd6a090637cb"
                                },
                                "BareMetalTDX": {
                                  "TrustedMeasurement": "b534c4fb9a7d4283d1030cfd71f00997f3b19f613171526b3e241abf92d3712d"
                                }
                              }
genpolicy version:            3.2.0.azl1.genpolicy0
```

Reply:
Sorry, I did not consider the underying tabwriter.